src/main/scala/akka/projection/testing/ConfigurablePersistentActor.scala

```diff
@@ -84,6 +86,9 @@ object ConfigurablePersistentActor {
                   ctx.self ! InternalPersist(totalEvents, testName, toPersist, bytesPerEvent, replyTo)
                 }
               }
+            case WakeUp(testName) =>
+              ctx.log.debug("WakeUp {}", ctx.self.path.name)
+              Effect.none
```

Comment:
Not important, but we are not using `testName`. We can use either log it or make `WakeUp` an object.